pyArango/database.py

```diff
@@ -195,6 +199,9 @@ def _checkCollectionList(lst):
             raise CreationError(data["errorMessage"], data)
         return self.graphs[name]
 
+    def createSatelliteGraph(self, name, createCollections = True, writeConcern = None):
+        return self.createGraph(name, createCollections, False, None, None, "satellite", None);
```

Comment:
writeConcern will be None regardless of the value entered as argument. I would also add the names of arguments explicitly as it makes it easier to spot these kind of bugs.